words/disasm.asm

```diff
@@ -402,7 +402,41 @@ disasm_jsr:
                 ; int>name returns zero if we just don't know.
                 lda 0,x
                 ora 1,x
+                bne _found_nt
+
+                ; So we didn't find the JSR target in the dictionary.
+                ; Check again at address-3 in case this is a JSR that
+                ; skipped underflow checking during compiling by adding
+                ; 3 to the JSR address.
+                lda scratch+1
+                sec
+                sbc #3         ; Subtract 3 this time.
+                sta 0,x
+                lda scratch+2
+                sbc #0         ; Subtract the carry if needed.
+                sta 1,x
+                ; ( xt )
+                jsr xt_int_to_name    ; Try looking again
+                ; int>name returns zero if we just don't know.
+                lda 0,x
+                ora 1,x
                 beq _no_nt
+
+                ; We got an nt this time.  Double check that it has underflow
```

Comment:
is there any case where we would find an NT at addr-3 where we wouldn't want to show the label?  ie. which would fail this sanity check?

Reply:
I gave that some thought, as I was originally going to skip the test.  There aren't any 3 byte words in Tali that I'm aware of, as it would need to be a two byte word with an RTS.  A quick scan through WORDLIST.md shows no words listed as 2 bytes (Tali doesn't count the RTS).

There could be in user code, though.  Here's an example:
```
assembler-wordlist >order

( A 3-byte word )
: undrop [ dex dex ] ;
( a target 3 bytes later with no name )
:noname 1234 ;
( a word to be disassembled )
: testword [ jsr ] ;   ( jsr uses address from :noname )

see testword ( jsr address is 3 bytes later than undrop )
```
There is also a version of Tali (admittedly heavily modified) that compiles headers separately from words (to make use of multiple paged banks where the dictionary lives in a different bank) where the situation would be more likely.

I believe, with the check, it always does the right thing.  It's also during disassembly where it's already done a slow dictionary lookup (through all wordlists) twice!  I'm not really worried about the speed or memory usage, and would feel better knowing it has correct behavior in all (even unusual) circumstances.

Also, just a quick note on c65 - it's very useful now.  I had an error in my branch logic and it was very easy to set a breakpoint by label and single step through just that section.  I do have one oddity, where I expected ENTER to repeat the previous instruction (which was `n` to single step) and it complains about an 'M' in the input.  Trying to repeat this now shows garbage in the input.  I'll open an issue for it.